earthpy/tests/test_stack_raster_tifs.py

```diff
@@ -21,3 +22,48 @@ def test_stack_raster_tifs(b1_b2_arrs):
     # Check Warning for Deprecation
     with pytest.raises(Warning, match="stack_raster_tifs is deprecated"):
         es.stack_raster_tifs([b1, b2], out_path="test.tif")
+
+
+def test_stack_raster(basic_image_tif):
```

Comment:
One minor suggestion here is to break this into multiple tests, each of which has one (ideally) assertion. One problem with writing tests like this is if one fails early (e.g., the first one), then none of the other things are tested. Another problem is that it requires the reader to remember a bunch of stuff about the execution context/environment 20 or 30 lines in, which can be hard.

Reply:
i agree with this. @mbjoseph i still can't find where tmpdir is defined. it looks like it should be a fixture but i can't find it to run the tests line by line locally.

Reply:
also feel free to review this but it just came up that - there is an unknown fixture !!!

Reply:
See https://docs.pytest.org/en/latest/tmpdir.html 

I'm happy to have you review it - I just stumbled across the PR and thought I'd Kool-Aid Man my way into the conversation.